testlib/src/test/java/io/spine/testing/TestsShould.java

```diff
@@ -217,22 +217,22 @@ private long now() {
         @Test
         @DisplayName("when values are equal")
         void equalValues() {
-            assertSecondsEqual(recentTime, recentTime, 0);
+            assertInDelta(recentTime, recentTime, 0);
         }
 
         @Test
         @DisplayName("when values are close")
         void closeValues() {
             // This method would be called within 10 seconds.
-            Tests.assertSecondsEqual(recentTime, now(), 10);
+            Tests.assertInDelta(recentTime, now(), 10);
```

Comment:
Not sure why you still use time. Please use regular `long` values. And there are way more cases:

* `expected` == `actual`;
* `expected` < `actual` and `actual` < (`expected` + delta);
* `expected` < `actual` and `actual`  == (`expected` + delta); 
* `expected` < `actual` and `actual`  >  (`expected` + delta) — negative scenario; 
* `expected` > `actual` and `actual` > (`expected` - delta);
* `expected` > `actual` and `actual`  == (`expected` - delta); 
* `expected` > `actual` and `actual`  < (`expected` - delta) — negative scenario.